app-conf/SchedulerConf.xml

```diff
@@ -39,6 +39,13 @@
     </scheduler>
 
     <scheduler>
+        <name>langoustine</name>
+        <classname>com.linkedin.drelephant.schedulers.LangoustineScheduler</classname>
```

Comment:
Langoustine is not open sourced yet.
It would be a better thing to put it in a separate namespace, as an optional Scheduler to Dr-Elephant.